<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>480</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QLabel" name="label_info">
      <property name="text">
       <string/>
      </property>
      <property name="alignment">
       <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="label_coord">
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
    <item>
     <layout class="QGridLayout" name="gridLayout">
      <item row="1" column="1">
       <widget class="QLCDNumber" name="lcdGNDSpeed"/>
      </item>
      <item row="0" column="0" colspan="4">
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <widget class="QLabel" name="label">
          <property name="text">
           <string>Heading</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLCDNumber" name="lcdDirection"/>
        </item>
        <item>
         <widget class="QLabel" name="label_2">
          <property name="text">
           <string>MagneticVariation</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLCDNumber" name="lcdMagVar"/>
        </item>
       </layout>
      </item>
      <item row="1" column="3">
       <widget class="QLCDNumber" name="lcdVerticalSpeed"/>
      </item>
      <item row="1" column="2">
       <widget class="QLabel" name="label_3">
        <property name="text">
         <string>VerticalSpeed(km/h)</string>
        </property>
       </widget>
      </item>
      <item row="2" column="3">
       <widget class="QLCDNumber" name="lcdVDOP"/>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="label_4">
        <property name="text">
         <string>GroundSpeed(km/h)</string>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="label_5">
        <property name="text">
         <string>HDOP(m)</string>
        </property>
       </widget>
      </item>
      <item row="2" column="2">
       <widget class="QLabel" name="label_6">
        <property name="text">
         <string>VDOP(m)</string>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="QLCDNumber" name="lcdHDOP"/>
      </item>
     </layout>
    </item>
    <item>
     <widget class="QDateTimeEdit" name="dateTimeEdit">
      <property name="displayFormat">
       <string>dd/MM/yyyy HH:mm:ss</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
